src/components/vaults/VaultMobileCard.tsx

```diff
@@ -55,91 +57,78 @@ export function VaultMobileCard (props: VaultMobileCardProps): JSX.Element {
         className='w-full'
         show={isOpen}
       >
-        {
-          props.vault.state === LoanVaultState.IN_LIQUIDATION
-            ? <LiquidatedVaultDetails />
-            : <ActiveVaultDetails vault={props.vault} />
-        }
+        <VaultMobileDetails vault={props.vault} />
       </Transition>
     </div>
   )
 }
 
-function ActiveVaultDetails (props: { vault: LoanVaultActive }): JSX.Element {
+function VaultMobileDetails (props: { vault: LoanVaultActive | LoanVaultLiquidated }): JSX.Element {
   return (
-    <div className='w-full mt-2 flex flex-col gap-y-1'>
-      <div className='w-full flex justify-between'>
-        Loans
-        <VaultTokenSymbols tokens={props.vault.loanAmounts} />
-      </div>
-      <div className='w-full flex justify-between'>
-        <div className='flex items-center gap-x-1 justify-end text-left'>
-          Loans Value (USD)
-          <InfoHoverPopover description='Loan token(s) and value (in USD) taken by a vault.' />
-        </div>
-        <NumberFormat
-          value={props.vault.loanValue}
-          displayType='text'
-          decimalScale={2}
-          fixedDecimalScale
-          thousandSeparator
-          prefix='$'
-        />
-      </div>
-      <div className='w-full flex justify-between'>
-        Collateral
-        <VaultTokenSymbols tokens={props.vault.collateralAmounts} />
-      </div>
-      <div className='w-full flex justify-between'>
-        <div className='flex items-center gap-x-1 justify-end text-left'>
-          Collateral Value (USD)
-          <InfoHoverPopover description='Type and value of tokens deposited as collaterals in a vault.' />
-        </div>
-        <NumberFormat
-          value={props.vault.collateralValue}
-          displayType='text'
-          decimalScale={2}
-          fixedDecimalScale
-          thousandSeparator
-          prefix='$'
-        />
-      </div>
-      <div className='w-full flex justify-between'>
-        <div className='flex items-center gap-x-1 justify-end text-left'>
-          Collateral Ratio
-          <InfoHoverPopover
-            description='Percentage of collaterals deposited in a vault in relation to the amount of loan taken.'
-          />
-        </div>
-        <span data-testid={`VaultRow.${props.vault.vaultId}.CollateralRatio`}>{`${props.vault.collateralRatio}%`}</span>
-      </div>
-    </div>
-  )
-}
+    <div className='w-full mt-2'>
+      <VaultDetailsListItem
+        title='Loans'
+        testId='VaultIdDetails.Loans'
+      >
+        {props.vault.state === LoanVaultState.IN_LIQUIDATION
```

Comment:
I will usually add `()` for ternary operations in JSX, it's easier to read after linting